README.md

```diff
@@ -3,30 +3,41 @@
 [![Build Status][build-badge]][build-link]
 [![Apache 2.0 License][license-badge]][license-link]
 
-The SDK has dependencies on specific versions of the C++ WebAssembly toolchain Emscripten (https://emscripten.org) and the protobuf library, therefor use of a Docker image is recommended.
+The SDK uses [bazel](https://bazel.build/) for its build. 
 
-## Docker
 
-A Dockerfile for the C++ SDK is provided in Dockerfile-sdk.
+## Creating a project for use
 
-It can built in this directory by:
+Create a directory with your WORKSPACE file, source files, and BUILD files:
+
+WORKSPACE file (WORKSPACE):
 
 ```
-docker build -t wasmsdk:v2 -f Dockerfile-sdk .
-```
+workspace(name = "my_extension")
 
-The docker image can be used for compiling wasm files.
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
-### Creating a project for use with the Docker build image
+# Point to local proxy wasm sdk. 
+# If you want to import remote proxy wasm sdk, use the `http_archive` rule below.
+local_repository(
+    name = "proxy_wasm_cpp_sdk",
+    path = "<path-to-local-proxy-cpp-sdk>",
+)
 
-Create a directory with your source files and a Makefile:
+# Replace '<CPP-SDK-SHA>' with actual SHA from github.
+# http_archive(
+#     name = "proxy_wasm_cpp_sdk",
+#     strip_prefix = "proxy-wasm-cpp-sdk-<CPP-SDK-SHA>",
+#     url = "https://github.com/bianpengyuan/proxy-wasm-cpp-sdk/archive/<CPP-SDK-SHA>.tar.gz",
+# )
 
-```
-PROXY_WASM_CPP_SDK=/sdk
+load("@proxy_wasm_cpp_sdk//bazel/dep:deps.bzl", "wasm_dependencies")
+
+wasm_dependencies()
 
-all: myproject.wasm
+load("@proxy_wasm_cpp_sdk//bazel/dep:deps_extra.bzl", "wasm_dependencies_extra")
 
-include ${PROXY_WASM_CPP_SDK}/Makefile.base_lite
+wasm_dependencies_extra()
 ```
 
 Source file (myproject.cc):
```

Comment:
This should probably be called `myplugin.cc`, not `myproject.cc`.

Reply:
Changed to my_plugin.cc